drivers/non_veterans_sub_pop/app/views/non_veterans_sub_pop/dashboards/non_veterans/_index_title.haml

```diff
@@ -1,7 +1,7 @@
 - content_for :title, 'Non-Veteran Dashboard'
-- @show_census = true
+- @show_census = false
 %h1= content_for :title
 - if @html
-  = render 'dashboards/tabs', active: 'non_veteran'
+  = render 'dashboards/tabs', active: 'non_veterans'
 %p.text-muted
-  Clients enrolled in homeless projects (ES, SH, SO, TH) where the client is not a veteran.
+  Clients enrolled in homeless projects (ES, SH, SO, TH) where the household has at least one adult (18+) and one child (< 18).
```

Comment:
"Clients enrolled in homeless projects (ES, SH, SO, TH) where the client is not a veteran."